src/miner.cpp

```diff
@@ -1225,13 +1514,22 @@ void StakeMiner(CWallet *pwallet)
 
         g_timer.GetTimes(function + "IsMiningAllowed", "miner");
 
+        CBlock StakeBlock;
+        std::map<GRC::Cpid, std::pair<uint256, GRC::MRC>> mrc_map;
+        std::map<GRC::Cpid, uint256> mrc_tx_map;
+
         LOCK(cs_main);
 
         g_timer.GetTimes(function + "lock cs_main", "miner");
 
         CBlockIndex* pindexPrev = pindexBest;
 
         // * Create a bare block
+
+        // This transition code is for mandatory change from V11 to v12 block format (accomodates MRC).
```

Comment:
typo, should be "accommodates"

Reply:
You are annoying!! But I love you anyway! Who else is going to catch these typos! :)